autovalet_system/scripts/slam.sh

```diff
@@ -13,13 +13,13 @@ then
             roslaunch autovalet_gazebo setup_slam_test.launch world_file:=state_machine_rturns_rpark.world x:=0 y:=6 1>> $HOME/slam_log.txt 2>&1
         else
             # right turns, left park
-            roslaunch autovalet_gazebo setup_slam_test.launch world_file:=state_machine_rturns_lpark.world x:=0 y:=6 1>> $HOME/slam_log.txt 2>&1
+            roslaunch autovalet_gazebo setup_slam_test.launch world_file:=right_right_left.world x:=0 y:=6 1>> $HOME/slam_log.txt 2>&1
         fi
     else
         if [ "$park" = "right" ]
         then
             # left turns, right park
-            roslaunch autovalet_gazebo setup_slam_test.launch world_file:=state_machine_lturns_rpark.world x:=-6 y:=-9 1>> $HOME/slam_log.txt 2>&1
+            roslaunch autovalet_gazebo setup_slam_test.launch world_file:=left_left_right.world x:=-6 y:=-9 1>> $HOME/slam_log.txt 2>&1
```

Comment:
why did we only  change two filenames? should keep it consistent whatever we choose..